apisix/plugins/redirect.lua

```diff
@@ -35,10 +36,26 @@ local schema = {
     properties = {
         ret_code = {type = "integer", minimum = 200, default = 302},
         uri = {type = "string", minLength = 2, pattern = reg},
+        regex_uri = {
+            description = "new uri that substitute from client uri " ..
```

Comment:
term "new uri" is not accurate to describe the `regex_uri` as it's actually a tuple that contains two elements.

Reply:
get it